opm/parser/eclipse/EclipseState/Schedule/MSW/SegmentSet.cpp

```diff
@@ -21,6 +21,11 @@
 #include <cmath>
 #include <map>
 
+#ifdef _WIN32
```

Comment:
OK - I have looked a bit more on this; and it seems it is the Gnu compiler which is non standard here. My suggestion is:
1. Remove the whole `#ifdef` block.
2. Introduce `static const double PI = 3.1415926535897932384626;`
3. Replace the `M_PI` with `PI`.